packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss-drilldown.vue

```diff
@@ -824,7 +825,7 @@ const setOutputSettingDefaults = () => {
 
 	if (!knobs.value.selectedInterventionVariables.length) {
 		knobs.value.interventionPolicyGroups.forEach((intervention) =>
-			selectedInterventionVariables.push(intervention.intervention.appliedTo)
+			selectedInterventionVariables.push(intervention.intervention.staticInterventions[0].appliedTo)
```

Comment:
Not every intervention policy group will have a static intervention.
Both of these are valid:
![image](https://github.com/user-attachments/assets/7b38268c-638e-4eca-a955-712c4a50b8d8)